handlers/leagues.js

```diff
@@ -43,7 +45,7 @@ var onLeaguesInMonthResponse = function onLeaguesInMonthResponse(message, callba
 
     if (response.eresult === 1) {
         if (this.debug) util.log("Received leagues in month response " + response.eresult);
-        this.emit("leaguesInMonthData", response.eresult, response);
+        this.emit("leaguesInMonthData", response.month, response.year, response.leagues);
```

Comment:
I think we should stick to the protobuf format. Even though you are filtering for eresult of 1, we should probably still return that 1, just for the sake of consistency.

Also, the default eresult seems to be 2. [eResult from the proto](https://github.com/RJacksonm1/node-dota2/blob/dev/proto/dota_gcmessages_client.proto#L1926)